framework/db/ConnectionInterface.php

```diff
@@ -0,0 +1,276 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\db;
+
+use yii\base\NotSupportedException;
+
+/**
+ * ConnectionInterface defines a connection to a database.
+ *
+ * @author Nelson J Morais <njmorais@gmail.com>
```

Comment:
API was originally authored by Qiang. I think his authoring copyright should not be completely removed.

```
@author Qiang Xue <qiang.xue@gmail.com>
```

Reply:
@SilverFire done! I removed my @author tag, can you review it if it's ok this way? thanks.